modules/igv/1.0/envs/crossmap.yaml

```diff
@@ -0,0 +1,56 @@
+name: crossmap
```

Comment:
Can you please check if this env can be re-used instead?
https://github.com/LCR-BCCRC/lcr-scripts/blob/master/envs/crossmap/convert_maf_coords.yaml

Reply:
I switched to using liftover so this env file is not needed anymore, I removed it from the module